source/mongoimport.txt

```diff
@@ -339,9 +331,7 @@ Options
 .. option:: --sslAllowInvalidCertificates
 
    Bypasses the validation checks for server certificates and allows
-   the use of invalid certificates. When using the
-   :setting:`~net.ssl.allowInvalidCertificates` setting, MongoDB logs as a
-   warning the use of the invalid certificate.
+   the use of invalid certificates. MongoDB logs a warning for invalid certificates.
```

Comment:
I don't think this change is correct. Its still supposed to link to the net.ssl.allowInvalidCertificates option, instead of the above.

Reply:
Fixed